runtime/ftplugin/groff.vim

```diff
@@ -0,0 +1,7 @@
+if exists('b:did_ftplugin')
```

Comment:
can you add a default header please?  Can you please also add yourself as maintainer to the MAINTAINERS file?